cmd/main.go

```diff
@@ -8,10 +8,16 @@ import (
 
 func main() {
 	srv, err := wallet.NewHTTPServer(8701, &wallet.Config{
-		Address:    "0xf8d6e0586b0a20c7",
-		PrivateKey: "f8e188e8af0b8b414be59c4a1a15cc666c898fb34d94156e9b51e18bfde754a5",
-		PublicKey:  "6e70492cb4ec2a6013e916114bc8bf6496f3335562f315e18b085c19da659bdfd88979a5904ae8bd9b4fd52a07fc759bad9551c04f289210784e7b08980516d2",
-		AccessNode: "http://localhost:8080",
+		Address:               "0xf8d6e0586b0a20c7",
+		PrivateKey:            "f8e188e8af0b8b414be59c4a1a15cc666c898fb34d94156e9b51e18bfde754a5",
+		PublicKey:             "6e70492cb4ec2a6013e916114bc8bf6496f3335562f315e18b085c19da659bdfd88979a5904ae8bd9b4fd52a07fc759bad9551c04f289210784e7b08980516d2",
+		AccountKeyId:          "0",
+		AccessNode:            "http://localhost:8888",
+		BaseUrl:               "http://localhost:8701",
+		ContractFungibleToken: "0xee82856bf20e2aa6",
+		ContractFlowToken:     "0x0ae53cb6e3f42a79",
+		ContractFUSD:          "0xf8d6e0586b0a20c7",
+		ContractFCLCrypto:     "0xf8d6e0586b0a20c7",
```

Comment:
Thinking we might deploy FCLCryptoContract on emulator start and update address refs throughout.
Looks like we're just using the FUSD address currently here and in `.env.example`

Prob in a new PR that also adds verify to embedded harness